pkgs/misc/vim-plugins/generated.nix

```diff
@@ -5920,17 +5920,6 @@ let
     };
   };
 
-  vim-which-key = buildVimPluginFrom2Nix {
-    pname = "vim-which-key";
-    version = "2020-03-16";
-    src = fetchFromGitHub {
-      owner = "liuchengxu";
-      repo = "vim-which-key";
-      rev = "4b70b443ba5838b165f90e26f9ed628d7d05e11d";
-      sha256 = "197rp20hngrq7qdlii0ai4vb81dlilah0q1wlil8hv3qcf4az7qr";
-    };
-  };
-
```

Comment:
@garbas can you please explain why a commit with message "aa" landed on nixpkgs and why it removes this vim plugin?

The commit message is, obviously, not self-explanatory!

Reply:
@matthiasbeyer the PR got squashed, so the commit message never made it to nixpkgs.

Reply:
Strange that github still shows the commit.